packages/nvidia-k8s-device-plugin/1100-k8s-device-plugin-all-device-patch.patch

```diff
@@ -0,0 +1,36 @@
+From ae5521585da70ff06ae66c9b02613cf85b5c3c0a Mon Sep 17 00:00:00 2001
+From: Jingwei Wang <jweiw@amazon.com>
+Date: Tue, 18 Mar 2025 23:23:40 +0000
+Subject: [PATCH] k8s-device-plugin all device patch
```

Comment:
Your commit message should be more descriptive about why the change is needed. That will also help us contribute the patch upstream.